### DISCUSSION TITLE
Idea for new command "/eat"

### DISCUSSION THREAD

**Discussion by AdmiralSky** [Zrips/CMI#6146] [Feature Requests]
In the process of creating a custom food, I am having a hard time figuring out how to manipulate Hunger and Saturation. (I can't create the same behavior with Saturation in Potion Effect as when I eat the food.)
So I thought how useful it would be if I could give [player] the same effect as when it eats a certain [food].
I propose a command of the form "/eat [player] [food]".

Sorry for the terrible English.

**Comment by mrfloris** [comment]
Personally I stick to things you can actually consume. The attachcommand lets you use the consume option. So commands only run when the item is actually eaten. 

I would like it indeed if we could create items that aren't food but give the eating effect visually and makes the sound and runs commands when the item is actually 'consumed`. I don't know how realistic it is to actually add that. But i sure like the idea.
